package.json

```diff
@@ -92,7 +92,6 @@
     "@vue/eslint-config-standard": "^5.1.2",
     "@vue/eslint-config-typescript": "^5.0.2",
     "@vue/test-utils": "^1.2.1",
-    "babel-core": "^7.0.0-bridge.0",
```

Comment:
yeah, it's this "breakage" stuff nodejs packages do that is a pita for these upgrades.  good job!